cmake/presets/x64-linux-clang.json

```diff
@@ -13,14 +13,18 @@
         "linux",
         "clang"
       ],
-      "displayName": "x64 linux clang static libs"
+      "displayName": "x64 linux clang static libs",
+      "cacheVariables": {
+        "CMAKE_CXX_FLAGS": "-stdlib=libc++"
```

Comment:
This flags should be moved to `src/cmake/set_cpp_pt_target_properties.cmake`.
presets are used only for developers and consumers use `src`.